components/create-lead/update-lead-form.tsx

```diff
@@ -79,20 +85,25 @@ const UpdateLeadForm = () => {
           meeting_notes: singleLeadData?.point_of_contact?.meeting_notes || '',
           reference: singleLeadData?.point_of_contact?.reference || '',
         },
-        reminder: [
-          {
-            title: 'ab_update',
-            user_id: parseInt(paramValue),
-            reminder_time: '2023-03-20T15:51:05+07:00',
-            notes: 'new_update',
-            status: 'pending',
-          },
-        ],
+        //! There is no option to update reminders in the UI so it is commented out for now but can be used in the future if needed to update reminders as well.
```

Comment:
Commented codes should removed into PR. Please clear all commented codes if those are no longer required.